Documentation/git-bash.txt

```diff
@@ -0,0 +1,51 @@
+git-bash(0)
+==========
+
+NAME
+----
+git-bash - A Windows initialiser for the bash window on that OS
```

Comment:
I am not sure that "initialiser" is a good term. Maybe `git-bash - Starts an interactive Bash in a new terminal window` instead?